### PR TITLE
gh-label: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.16.1

### CODE REVIEW

File: pages/common/gh-label.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Delete a label for the repository in the current directory prompting for confirmation:
+
+`gh label delete {{name}}`
+
+- Update name and description for a specific label for the repository in the current directory:
+
+`gh label edit {{name}} --name {{new_name}} --description "{{description}}"`
+
+- Clone labels from a specific repository into the repository in the current directory:
```
Comment (mfrw) [inline pages/common/gh-label.md:26]: How about ?
```suggestion
- Clone labels from a specific repository for the repository in the current directory:
```

**Review by mfrw — APPROVED** [review]
LGTM with a suggestion

File: pages/common/gh-label.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/gh-label.md:26]: I think **into** fits the context better here.

**Review by kbdharun — APPROVED** [review]
LGTM

File: pages/common/gh-label.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`gh label clone {{owner}}/{{repository}}`
```
Comment (blueskyson) [inline pages/common/gh-label.md:28]:
```suggestion
- Clone labels from a source repository into the destination repository (current directory by default):

`gh label clone {{owner}}/{{source_repository}}`
```

Comment (kbdharun) [reply pages/common/gh-label.md:28]:
```suggestion
- Clone labels from the source repository into the destination repository in the current directory:

`gh label clone {{owner}}/{{repository}}`
```
Would this better?

File: pages/common/gh-label.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- View labels for the repository in the current directory in the default web browser:
+
+`gh label list --web`
+
+- Create a label with a specific name, description and color in hexadecimal format for the repository in the current directory:
+
+`gh label create {{name}} --description "{{description}}" --color {{color_hex}}`
+
+- Delete a label for the repository in the current directory prompting for confirmation:
```
Comment (CleanMachine1) [inline pages/common/gh-label.md:18]:
```suggestion
- Delete a label for the repository in the current directory, prompting for confirmation:
```

File: pages/common/gh-label.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`gh label delete {{name}}`
+
+- Update name and description for a specific label for the repository in the current directory:
```
Comment (CleanMachine1) [inline pages/common/gh-label.md:22]:
```suggestion
- Update the name and description for a specific label for the repository in the current directory:
```

File: pages/common/gh-label.md (hunk 2 above)
Comment (CleanMachine1) [inline pages/common/gh-label.md:28]:
```suggestion
`gh label clone {{slug}}`
```

Comment (navarroaxel) [reply pages/common/gh-label.md:28]: We use `{{owner}}/{{repository}}` is several gh pages, `gh-issue`, `gh-extension`, etc.

Comment (CleanMachine1) [reply pages/common/gh-label.md:28]: :+1:

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**